src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -3477,4 +3477,36 @@ WindowID MessageHelper::ExtractWindowIdFromSmartObject(
   return mobile_apis::PredefinedWindows::DEFAULT_WINDOW;
 }
 
+void MessageHelper::RemoveEmptyMessageParams(
+    smart_objects::SmartObject& msg_params) {
+  if (msg_params.empty()) {
+    return;
+  }
+
+  for (const auto& item : msg_params.enumerate()) {
+    auto& param = msg_params[item];
+
+    if (param.empty()) {
+      msg_params.erase(item);
+      break;
```

Comment:
Seems need continue here instead break

Reply:
this also applies to other `breaks` in this loop

Reply:
done